tvplayer/src/main/java/com/kaltura/tvplayer/playlist/BasicMediaOptions.java

```diff
@@ -0,0 +1,33 @@
+package com.kaltura.tvplayer.playlist;
+
+import com.kaltura.playkit.PKMediaEntry;
+import com.kaltura.playkit.PKPlaylistMedia;
+
+public class BasicMediaOptions extends PKPlaylistMedia {
+
+    private PKMediaEntry pkMediaEntry;
+    private CountDownOptions countDownOptions;
```

Comment:
- rename to playlistCountDownOptions -- everywhere
- mark as Nullable -- everywhere

Reply:
done